src/status_im/mailserver/core.cljs

```diff
@@ -249,7 +251,10 @@
     (fx/merge cofx
               {:db (cond-> (dissoc db :mailserver/current-request)
                      gap-request?
-                     (assoc :mailserver/fetching-gaps-in-progress {}))}
+                     (as-> $
```

Comment:
This looks a bit odd, some leftover? you probably can thread first directly without `as->` or I am missing something?